components/bitcoin-topics/Navigation.tsx

```diff
@@ -40,7 +48,55 @@ export function Navigation({
     onLinkClick?: React.MouseEventHandler<HTMLAnchorElement>
 }) {
     const pathname = usePathname()
-    const posts = allCoreContent(sortPosts(allTopics))
+    const posts = useMemo(() => allCoreContent(sortPosts(allTopics)), [])
+    const [expandedTopics, setExpandedTopics] = useState<
+        Record<string, boolean>
+    >({})
+    const [lastVisitedTopic, setLastVisitedTopic] = useState<string | null>(
```

Comment:
lastVisitedTopic is not used.

Reply:
removed here https://github.com/bitcoin-dev-project/bitcoin-dev-project/pull/102/commits/16c40f9445f3a96c83a9e54b8e91235136e27a20